packages/api/amplify_api/lib/src/graphql/ws/web_socket_connection.dart

```diff
@@ -203,12 +391,15 @@ class WebSocketConnection implements Closeable {
 
   /// Times out the connection (usually if a keep alive has not been received in time).
   void _timeout(Duration timeoutDuration) {
+    _timeoutTimer?.cancel();
```

Comment:
Is the timeout timer still running when we lose network? Should it be?

Reply:
Currently no, the timer gets canceled during network interruptions. I think this is how we want it, do you think otherwise?